### ISSUE TITLE
Missing projects from some installed plugin in the model library 

### ISSUE THREAD

**Issue by RoiArthurB** [gama-platform/gama.old#3886]
**Describe the bug**
The plugin `physics` doesn't add models in the `Plugin model` folder after a new workspace initialization on Linux 

**To Reproduce**
1. Install latest GAMA
2. Create new workspace
3. Wait for the initialization to finish
4. See 13 projects instead of 14 on `Plugin model` folder

**Expected behavior**
Have every plugin's demo model added in library

**Screenshots**

![image](https://github.com/gama-platform/gama/assets/16764085/f066221a-dbed-4399-9a0e-0fc6ab7314ae)

![image](https://github.com/gama-platform/gama/assets/16764085/84c1e240-6f2f-4526-8271-f1432c878d29)

![image](https://github.com/gama-platform/gama/assets/16764085/7e947aba-ad30-404a-b378-124e075a3554)

**Desktop (please complete the following information):**
 - OS: Linux
 - GAMA version: 1.9.2
 - Java version: 17.0.8.1

---

EDIT : While manually copying `Stairs.gaml` model, it works perfectly fine, it's just the model library which is missing :/

![Stairs_model_display_Climb_cycle_233262_time_1695289796556](https://github.com/gama-platform/gama/assets/16764085/63b5db1e-685e-41ac-9203-956a4539749c)

**Comment by AlexisDrogoul** [comment]
When exploring `Gama.app` on macOS, I can see that the `models` folder inside `Contents/Eclipse/plugins/simtools.gaml.extensions.physics_1.9.2.202309031322` is effectively empty. So it means that the problem occurred at compilation / assemblage time, while https://github.com/gama-platform/gama/tree/GAMA_1.9.2/simtools.gaml.extensions.physics/models/Physics%20Engine is obviously not...

**Comment by RoiArthurB** [comment]
But do you have the `Physics` folder in GAMA on MacOS ? Because, what's odd, is that it's missing on Linux but present on Windows 11... :thinking:

**Comment by AlexisDrogoul** [comment]
Missing on macOS too. `models` is empty in the plugin and so no models is copied in the workspace of GAMA.

**Comment by hqnghi88** [comment]
You mean 1.9.2 release? i download withJDK_M1 and it is here with physics plugins, in a new GAMA workspace
<img width="725" alt="Screenshot 2023-09-22 at 16 18 24" src="https://github.com/gama-platform/gama/assets/6105384/888631f1-4314-4f13-9f4d-5106b5485a64">

**Comment by AlexisDrogoul** [comment]
Absent from GAMA_1.9.2_MacOS_M1.dmg as you can see below. 

<img width="376" alt="plugins 2023-09-23 09-09-07" src="https://github.com/gama-platform/gama/assets/579256/22eb388c-d356-49f0-b3de-aab54c7a4521">

But present in the one with JDK ! See below: 

<img width="384" alt="Gama 1 9 2 with JDK app 2023-09-23 09-11-59" src="https://github.com/gama-platform/gama/assets/579256/e533a4b1-6511-42fd-9738-00d19bb3af80">


Thank you @hqnghi88, you nailed it !

@RoiArthurB it is definitely an "assemblage" problem during the compilation on GHA.

**Comment by RoiArthurB** [comment]
Make no sense, the jdk version is just a copy-pasted project (with add/modified file to embed JDK), so it shouldn't have any difference between versions for those files.

Also, this folder is properly filled after compilation for each os
![image](https://github.com/gama-platform/gama/assets/16764085/1f7f4a6a-1330-4d48-91a0-4d8a144bc3c4)

**Comment by RoiArthurB** [comment]
Tested and fixed on c2e0cc20 for Linux w/o JDK :) 

```
plugins/simtools.gaml.extensions.physics_1.9.3.202309250401/models
└── Physics Engine
    ├── images
    │   ├── ball.jpg
    │   ├── bam.png
    │   ├── bang.png
    │   ├── basket.jpg
    │   ├── DEM
    │   │   ├── DEM.jpg
    │   │   ├── DEM.png
    │   │   ├── RedRiver.asc
    │   │   ├── Volcano DEM.asc
    │   │   ├── Volcano DEM.asc.aux.xml
    │   │   ├── Volcano DEM.prj
    │   │   └── Volcano Texture.jpg
    │   ├── foot.jpg
    │   ├── marble2.jpg
    │   ├── marble.jpg
    │   ├── mat.jpg
    │   ├── tennis.jpg
    │   └── wood.jpg
    └── models
        ├── Eroding Vulcano.gaml
        ├── Falling.gaml
        ├── Flow on Terrain.gaml
        ├── Perfect Gas Chamber.gaml
        ├── Play Pool.gaml
        ├── Stairs.gaml
        ├── Testing Restitution.gaml
        ├── Testing Steps.gaml
        └── Tricky Fountain.gaml

5 directories, 26 files
```
